### PR TITLE
Fix Vue test: Common-StylingMode

### PR DESCRIPTION
**Pull request by jacksonrr3**

### CODE REVIEW

**Review by nightskylark — DISMISSED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by vconst — DISMISSED** [review]

File: JSDemos/Demos/Common/StylingMode/Angular/app/app.component.css [hunk 1]
```diff
@@ -8,13 +8,20 @@
     margin-right: 320px;
 }
 
-::ng-deep .editors .left, ::ng-deep .editors .right {
+::ng-deep .editors .left {
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/Angular/app/app.component.css]: Try this
```suggestion
::ng-deep .editors .left, ::ng-deep .editors .right{
```

File: JSDemos/Demos/Common/StylingMode/Angular/app/app.component.css [hunk 2]
```diff
@@ -8,13 +8,20 @@
...
     box-sizing: border-box;
 }
 
+::ng-deep .editors .right {
+    display: inline-block;
+    width: 49%;
+    padding-right: 27px;
+    box-sizing: border-box;
+}
+
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/Angular/app/app.component.css]: And this
```suggestion
::ng-deep .editors .right {
    padding-right: 27px;
}

```

File: JSDemos/Demos/Common/StylingMode/AngularJS/styles.css [hunk 3]
```diff
@@ -8,13 +8,20 @@
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/AngularJS/styles.css]: The same as above

File: JSDemos/Demos/Common/StylingMode/React/styles.css [hunk 4]
```diff
@@ -8,14 +8,21 @@
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/React/styles.css]: The same as above

File: JSDemos/Demos/Common/StylingMode/Vue/App.vue [hunk 5]
```diff
@@ -123,15 +125,18 @@ export default {
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/Vue/App.vue]: The same as above

File: JSDemos/Demos/Common/StylingMode/jQuery/styles.css [hunk 6]
```diff
@@ -8,13 +8,20 @@
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline JSDemos/Demos/Common/StylingMode/jQuery/styles.css]: The same as above

File: MVCDemos/Content/DemosStyles/Common/StylingMode.css [hunk 7]
```diff
@@ -8,13 +8,20 @@
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline MVCDemos/Content/DemosStyles/Common/StylingMode.css]: The same as above

File: NetCoreDemos/wwwroot/css/DemosStyles/Common/StylingMode.css [hunk 8]
```diff
@@ -8,13 +8,20 @@
     margin-right: 320px;
 }
 
-.editors .left, .editors .right {
+.editors .left {
```
Comment (alexserov) [inline NetCoreDemos/wwwroot/css/DemosStyles/Common/StylingMode.css]: The same as above

**Review by alexserov — DISMISSED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by alexserov — APPROVED** [review]

**Pull request merged**